Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG PYTHON_VERSION=3.8
+FROM python:${PYTHON_VERSION} AS base
+
+WORKDIR /app
+
+COPY . .
+
+FROM python:${PYTHON_VERSION}-slim
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY --from=base /app .
+
+RUN pip install --no-cache-dir -r requirements.txt && \
+    python manage.py migrate
+
+EXPOSE 8080
+
+ENTRYPOINT ["python", "manage.py", "runserver", "0.0.0.0:8080"]
```

Comment:
![image](https://github.com/user-attachments/assets/dd86f4af-62c8-4e20-b6e1-53eb9e3dc791)
Add Eof(End-of-file), need to add a blank line at the end

Reply:
Oh, sure
EOF added

Reply:
Thanks!